DirectEmployers/MyJobs
# [PD-3692] analytics cleanup work

## What's this PR do?
PR just does a little cleanup work on analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any of the analytics apps

## How should this be manually tested?
View the code and Pull down the repository. Check the analytics and make sure it is still working with the clicking on the tabs, filters, and table filters

## Any background context you want to provide?
PR was originally just to remove some code and do a css change on the table but included some other minor cleanup work that needed to be done anyways
